docs/tools/developer-tools/quickstart/advanced-usage/container.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: Container
+description: Quickstart is packaged as a docker container
+sidebar_label: Container
+sidebar_position: 10
+---
+
+# Container
+
+Quickstart is packaged as a docker container image and provides a default, non-validating, ephemeral configuration that should work for most developers. By configuring a container using this image with a host-based volume (described below in the "Configuration" section) an operator gains access to full configuration customization and persistence of data.
+
+The Quickstart [docker image](https://hub.docker.com/r/stellar/quickstart) uses the following software:
+
+- [PostgreSQL](https://www.postgresql.org)
+- [stellar-core](https://github.com/stellar/stellar-core)
+- [horizon](https://github.com/stellar/go/tree/master/services/horizon)
+- [friendbot](https://github.com/stellar/go/tree/master/services/friendbot)
+- [stellar-rpc](https://github.com/stellar/stellar-rpc/tree/main/cmd/stellar-rpc)
+- [supervisord](http://supervisord.org/)
+
+The Stellar CLI has commands for starting and stopping a Quickstart container, and also a command to get logs from a running container. See Stellar CLI [documentation](https://developers.stellar.org/docs/tools/developer-tools/cli/stellar-cli#stellar-container) for more information.
```

Comment:
let's change this link to a relative link `../../to/the/file.mdx#anchor` instead of the URL